analytics/src/main/java/edu/artic/analytics/AnalyticsTracker.kt

```diff
@@ -23,23 +27,33 @@ interface AnalyticsTracker {
 }
 
 class AnalyticsTrackerImpl(context: Context,
+                           private val languageSelector: LanguageSelector,
+                           private val membershipPrefs: MemberInfoPreferencesManager,
                            private val analyticsConfig: AnalyticsConfig) : AnalyticsTracker {
 
     private val analytics = GoogleAnalytics.getInstance(context)
     private val tracker = analytics.newTracker(analyticsConfig.trackingId)
 
     override fun reportEvent(category: String, action: String, label: String) {
+        val memberId = membershipPrefs.memberID
         tracker.send(HitBuilders.EventBuilder()
                 .setCategory(category)
                 .setAction(action)
+                .setCustomDimension(2, (if (memberId != null) "Member" else "None"))
+                .setCustomDimension(3, languageSelector.getAppLocale().nameOfLanguageForAnalytics())
+                .setCustomDimension(4, Locale.getDefault().toString())
```

Comment:
Shouldn't this be `Locale.getDefault().toLanguageTag()` instead? I'm assuming we need to use the BCP 47 format, not the backward-compatible java String (c.f. javadoc on `Locale::toString`).

Reply:
which toString offers

Reply:
The requirements gave `en-US` as an example...I've left a comment over there, hopefully we can sort that out in the AM.

I'll accept this PR with these two locales in either format for now.